diff --git a/CMakeLists.txt b/CMakeLists.txt
index 41c1f85a..c06c8938 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -51,10 +51,10 @@ else()
 	message("-- INFO: Using VCPKG if detected")
 endif()
 
-if (WIN32 AND NOT MINGW AND BUILD_SHARED_LIBS)
-	message("-- INFO: Configuring .rc resource script")
-	configure_file("${CMAKE_CURRENT_SOURCE_DIR}/src/dpp/dpp.rc.in" "${CMAKE_CURRENT_SOURCE_DIR}/src/dpp/dpp.rc" NEWLINE_STYLE WIN32)
-endif()
+# if (WIN32 AND NOT MINGW AND NOT CMAKE_CXX_COMPILER_FRONTEND_VARIANT MATCHES GNU AND BUILD_SHARED_LIBS)
+# 	message("-- INFO: Configuring .rc resource script")
+# 	configure_file("${CMAKE_CURRENT_SOURCE_DIR}/src/dpp/dpp.rc.in" "${CMAKE_CURRENT_SOURCE_DIR}/src/dpp/dpp.rc" NEWLINE_STYLE WIN32)
+# endif()
 
 if (NOT DPP_NO_VCPKG AND EXISTS "${_VCPKG_ROOT_DIR}")
 	set(PROJECT_NAME "dpp")
@@ -65,7 +65,7 @@ if (NOT DPP_NO_VCPKG AND EXISTS "${_VCPKG_ROOT_DIR}")
 		HOMEPAGE_URL "https://dpp.dev/"
 		DESCRIPTION "An incredibly lightweight C++ Discord library."
 	)
-	add_subdirectory(library-vcpkg)	
+	add_subdirectory(library-vcpkg)
 else()
 	set(PROJECT_NAME "libdpp")
 	project(
diff --git a/cmake/CPackSetup.cmake b/cmake/CPackSetup.cmake
index 75e914db..e17c1a84 100644
--- a/cmake/CPackSetup.cmake
+++ b/cmake/CPackSetup.cmake
@@ -17,7 +17,7 @@ if (WIN32)
 		ARCHIVE DESTINATION  ${DPP_INSTALL_LIBRARY_DIR}
 		RUNTIME DESTINATION  ${CMAKE_INSTALL_BINDIR}
 		INCLUDES DESTINATION ${DPP_INSTALL_INCLUDE_DIR})
-	install(DIRECTORY "${CMAKE_SOURCE_DIR}/include/" DESTINATION "${DPP_INSTALL_INCLUDE_DIR}")
+	install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/../include/" DESTINATION "${DPP_INSTALL_INCLUDE_DIR}")
 else()
 	install(TARGETS dpp
 		EXPORT ${DPP_EXPORT_NAME}
@@ -34,7 +34,7 @@ write_basic_package_version_file(${DPP_VERSION_FILE}
 		COMPATIBILITY SameMajorVersion)
 
 ## Include the file which allows `find_package(dpp)` to function.
-install(FILES "${CMAKE_SOURCE_DIR}/cmake/dpp-config.cmake" "${DPP_VERSION_FILE}" DESTINATION "${DPP_CMAKE_DIR}")
+install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/../cmake/dpp-config.cmake" "${DPP_VERSION_FILE}" DESTINATION "${DPP_CMAKE_DIR}")
 
 ## Export the targets to allow other projects to easily include this project
 install(EXPORT "${DPP_EXPORT_NAME}" DESTINATION "${DPP_CMAKE_DIR}" NAMESPACE dpp::)
